spring-cloud-circuitbreaker-resilience4j/src/main/java/org/springframework/cloud/circuitbreaker/resilience4j/Resilience4JCircuitBreakerFactory.java

```diff
@@ -114,6 +121,18 @@ public Resilience4JCircuitBreaker create(String id) {
 				Optional.ofNullable(circuitBreakerCustomizers.get(id)), bulkheadProvider);
 	}
 
+	@Override
+	public Resilience4JCircuitBreaker create(String id, String groupName) {
```

Comment:
Could we find a way to reuse code between the two create methods?

Reply:
Changed